examples/capi-quick-start/nutanix-cluster-calico-crs.yaml

```diff
@@ -67,7 +67,7 @@ spec:
             credentials:
               secretRef:
                 name: ${CLUSTER_NAME}-pc-creds
-            strategy: HelmAddon
+            strategy: ClusterResourceSet
```

Comment:
Please revert all these example cluster changes and rebase on main now https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/765 is merged.